types/env_vars.pp

```diff
@@ -0,0 +1,14 @@
+type Sensu::Env_Vars = Struct[{
+
+  Optional[EMBEDDED_RUBY] => Boolean,
```

Comment:
Wow, love the type validation here of the struct!

Reply:
yeah, struct is the missing link to allow usage of single hashes to manage AND validate multiple configuration settings without multiplying classes' parameters.
Only limit is that currently, afaik, it' not possible to allow any possible key value, the ones accepted, even if optional, must be defined in the struct.